picard/util/tags.py

```diff
@@ -166,652 +227,1128 @@ def names(self, selector=None):
         shortdesc=N_('FIXME:recording_firstreleasedate'),
+        longdesc=N_('The date of the earliest recording for a track in the format YYYY-MM-DD.'),
         is_hidden=True,
         is_tag=False,
     ),
+    TagVar(
+        'recording_series',
+        shortdesc=N_('FIXME:recording_series'),
+        longdesc=N_('A multi-value variable containing the series titles associated with the recording.'),
+        is_hidden=True,
+    ),
+    TagVar(
+        'recording_seriescomment',
+        shortdesc=N_('FIXME:recording_seriescomment'),
+        longdesc=N_('A multi-value variable containing the series disambiguation comments associated with the recording.'),
+        is_hidden=True,
+    ),
+    TagVar(
+        'recording_seriesid',
+        shortdesc=N_('FIXME:recording_seriesid'),
+        longdesc=N_('A multi-value variable containing the series MusicBrainz Identifiers (MBIDs) associated with the recording.'),
+        is_hidden=True,
+    ),
+    TagVar(
+        'recording_seriesnumber',
+        shortdesc=N_('FIXME:recording_seriesnumber'),
+        longdesc=N_('A multi-value variable containing the series numbers associated with the recording.'),
+        is_hidden=True,
+    ),
     TagVar(
         'recordingcomment',
         shortdesc=N_('FIXME:recordingcomment'),
+        longdesc=N_('The disambiguation comment for the recording associated with a track.'),
         is_hidden=True,
         is_tag=False,
     ),
     TagVar(
         'recordingtitle',
         shortdesc=N_('FIXME:recordingtitle'),
+        longdesc=N_('Recording title - normally the same as the track title, but can be different.'),
+        is_hidden=True,
+        is_tag=False,
+    ),
+    TagVar(
+        'releaseannotation',
+        shortdesc=N_('FIXME:releaseannotation'),
+        longdesc=N_('The annotation comment for the release.'),
         is_hidden=True,
         is_tag=False,
     ),
     TagVar(
         'releasecomment',
         shortdesc=N_('FIXME:releasecomment'),
+        longdesc=N_('The disambiguation comment for the release.'),
         is_hidden=True,
         is_tag=False,
     ),
     TagVar(
         'releasecountries',
         shortdesc=N_('FIXME:releasecountries'),
+        longdesc=N_('A multi-value variable containing the complete list of release countries for the release.'),
         is_hidden=True,
         is_tag=False,
     ),
+    TagVar(
+        'releasecountry',
+        shortdesc=N_('Release Country'),
+        longdesc=N_(
+            'The two-character code for the country in which the release was issued. If more than one release country '
+            'was specified, this tag will contain the first one in the list.'
+        ),
+    ),
+    TagVar(
+        'releasedate',
+        shortdesc=N_('Release Date'),
+        # TODO: Confirm long description and that this is not provided from the MB data.
+        longdesc=N_(
+            'The date that the release (album) was issued, in the format YYYY-MM-DD.'
```

Comment:
```suggestion
            'The date that the release (album) was issued, in the format `YYYY-MM-DD`.'
```

Not sure about that, but I think date formats should be enclosed in back quotes.